apps/rush/src/taskRunner/ProjectBuildTask.ts

```diff
@@ -229,17 +229,19 @@ export default class ProjectBuildTask implements ITaskDefinition {
       throw new Error(`The project [${this._rushProject.packageName}] does not define a 'test' or 'build' command in the 'scripts' section of its package.json`);
     }
 
-    // Normalize test command step
-    build.args.push(this._errorDisplayMode === ErrorDetectionMode.VisualStudioOnline ? '--no-color' : '--color');
+    if (build.args) {
```

Comment:
maybe we should add a package to web build tools that has an empty build, just to make sure it's a supported scenario. Just an idea.

Also, add a change file I believe?